docs/src/user-guide/struct-logs/index.md

```diff
@@ -0,0 +1,21 @@
+# Overview
+
+The following are guides for working with structured logs.
+
+::::{grid} 1 1 2 2
+:gutter: 2
+
+:::{grid-item-card}
+:link: format/index
+Formatting as plain text
+^^^
+A guide to formatting logs as plain text with a format string.
```

Comment:
```suggestion
How to format structured logs as plain text, using a format string.
```

"A guide to" feels redundant. I know this breaks the convention we have in the other docs, but I'd prefer to start making the improvement here.